helm-charts/falcon-sensor/README.md

```diff
@@ -29,10 +29,8 @@ The Falcon Helm chart has been tested to deploy on the following Kubernetes dist
 
 | Helm chart Version      | Falcon Sensor Version             |
 |:------------------------|:----------------------------------|
-| `<= 1.6.x`              | `<= 6.34.x`                       |
-| `>= 1.7.x && <= 1.17.x` | `>= 6.35.x && < 6.49.x`           |
-| `>= 1.18.x`             | `>= 6.49.x`                       |
-| `>= 1.19.x`             | `>= 6.54.x`                       |
```

Comment:
For my clarification; We removed the old versions, since those sensor versions are not supported?

Reply:
Correct, yes.